visualization_tool/src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,70 @@
+import {useEffect} from 'react';
+import IconButton from '@mui/material/IconButton';
+import {KeyboardCommandKey} from '@mui/icons-material';
+
+import './Navbar.css';
+import logo from '/logo.png';
+
+type NavbarProps = {
+  searchQuery: string;
+  setSearchQuery: React.Dispatch<React.SetStateAction<string>>;
+};
+
+const Navbar = ({searchQuery, setSearchQuery}: NavbarProps) => {
+  useEffect(() => {
+    const searchInput = document.getElementById(
+      'search-input'
+    ) as HTMLInputElement;
+    // autofocus on search input
+    searchInput.focus();
+    // focus on search input with ctrl + f
+    const handleKeyDown = (e: KeyboardEvent) => {
+      if (e.ctrlKey && e.key === 'f') {
+        e.preventDefault();
+        searchInput.focus();
+      }
+    };
+    document.addEventListener('keydown', handleKeyDown);
+    return () => {
+      document.removeEventListener('keydown', handleKeyDown);
+    };
+  }, []);
+
+  return (
+    <>
+      <header>
+        <img src={logo} className="logo" alt="GCP Scanner logo" />
+        <p>Scanner</p>
+      </header>
+      <div className="pages-nav">
+        <div className="links">
+          <a className="active" href="">
+            Resources
+          </a>
+          <a href="">IAM Policy</a>
```

Comment:
I know it was already reviewed and approved but why do we have IAM Policy as a separate tab?

Reply:
This is just a place holder for other tabs in the feature and to show the design
I think we can remove it now